docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -159,3 +148,24 @@ Currently, there are not many tools for testing. The APB team has a few open Git
 
 * link:https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/issues/131[lint command for APB content]
 * link:https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/docs/proposals/testing.md[CI and Image tests]
+
+== Automated Builds
+
+Every `aerogear` APB has a Docker Hub repository set up for hosting images.
+When a PR for an APB repository is merged to master, the Docker Hub respository detects this and kicks off an automated build.
+If the build is successful, the resulting image is tagged as `latest`, replacing the previous `latest` image for that APB.
+
+=== Setting up an Automated Build for APB Images
+
+IMPORTANT: Docker repositories for APBs need to be created as an 'Automated Build' rather than a 'Repository'. There doesn't seem to be a way to add an automated build to a repository afterwards.
+
+While signed into Docker Hub from a browser:
```

Comment:
This seems to be generic so maybe we can just point to docker documentation?

Reply:
Got a link?
I couldn't find docs for this, which is why I added this section :)

Reply:
+1 on keeping this in. It's small and it's less likely someone will read it if we link to it externally